src/checks.py

```diff
@@ -98,6 +98,16 @@ def check_num_cpus() -> Tuple[bool, str, str]:
   output = run_command(["cat /proc/cpuinfo | grep processor | wc -l"])
   return (True, output, None) if int(output) >= MINIMUM_CPU_THREADS else (False, output, "sui-node requires >= 48 CPU threads")
 
+def check_cpu_speed() -> Tuple[bool, str, str]:
+  output = run_command(["./check_cpu_speed"], "lib/bin")
```

Comment:
can we have the program print out the results and then do the comparisons in python? will make it easier to update limits, plus we'll eventually probably want to gather them all in one place.